Automattic/wp-calypso
# Support: Update URL for map existing domain doc

Fixes #56119.

#### Proposed Changes

* Updates map-existing-domain URL to using the correct one.

#### Testing Instructions



* Go to https://container-modest-jones.calypso.live/domains/manage/SITE.
* In the top right click "Add a domain".
* Select "Use a domain I own".
* Enter `lanyards.obenland.it` and submit.
* Under "Connect your domain", inspect the "learn more" link and make sure it links to `https://wordpress.com/support/domains/map-existing-domain/`

<img width="1374" alt="Screen Shot 2022-10-18 at 11 44 36 AM" src="https://user-images.githubusercontent.com/1398304/196519560-a0a2067c-2ae8-40a2-83d4-7f34e8079444.png">


#### Pre-merge Checklist






Fixes #56119.
